### ISSUE TITLE
[package] gsl-lite/0.34.0: Conanfile format error on recent versions of Conan

### ISSUE THREAD

**Issue by parkertomatoes** [conan-io/conan-center-index#4726]
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **gsl-lite/1.2.8**
  * Operating System+version: **Linux Ubuntu 18.04** (WSL1 Windows 10 1809)
  * Compiler+version: **GCC 8**
  * Conan version: **conan 1.34.0**
  * Python version: **Python 3.6.9**

### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
[settings]
os=Linux
os_build=Linux
arch=x86_64
arch_build=x86_64
compiler=gcc
compiler.version=7
compiler.libcxx=libstdc++
build_type=Release
[options]
[build_requires]
[env]
```

### Steps to reproduce (Include if Applicable)
Create a conanfile.txt with the following:
```
[requires]
gsl-lite/0.34.0
```
Then run conan install. This works fine on Conan 1.30.2. On Conan 1.34.0 it fails with:
>```
>ERROR: .../.conan/data/gsl-lite/0.34.0/_/_/export/conanfile.py: Recipe 'description' must be a string.
>```
Sure enough, the conanfile for the package has a tuple for its description:
>```python
> description = ("A single-file header-only version of ISO C++ ",
>                   "Guideline Support Library (GSL) for C++98, C++11 and later")
>```

The restriction seems to have been added with this PR:
https://github.com/conan-io/conan/pull/8143

**Comment by uilianries** [comment]
Related to https://github.com/conan-io/conan/issues/8572

**Comment by SpaceIm** [comment]
Remove `gsl-lite/0.34.0` from your cache, and reinstall. You have an old revision of this recipe.

**Comment by uilianries** [comment]
Fixed by #3752
